docs/cvl/methods.md

```diff
@@ -444,14 +444,16 @@ contains the address of the receiver contract of the summarized call.
 
 There are a few restrictions on the functions that can be used as approximations:
 
- - Functions used as summaries are not allowed to call contract functions.
-
  - The types of any arguments passed to or values returned from the summary
    must be {ref}`convertible <type-conversions>` between CVL and Solidity types.
    Arguments that are not accessed in the summary may have any type.
+  
+ - In case of recursive calls due to the summarization, the recursion limit can be set with 
+   `--prover_args '-contractRecursionLimit N'` where `N` is the number of recursive calls allowed (default 0).
 
 Function summaries for *internal* methods have a few additional restrictions on
```

Comment:
this is no longer a list of restrictions only on argument and return types

Reply:
what I meant is that the restriction you added to the list on line 455 is not an argument or return-type limitation, so this "title" of the list needs rephrasing.

Reply:
In the new version this isn't mentioned as a restriction. First because it's as you said,  unrelated to args or return types, and therefore the original list is now not a list. Or am I missing something

Reply:
ah wait, you referrd to the section on internal functions